tests/components/zha/common.py

```diff
@@ -70,12 +73,41 @@ def unique_id(self):
 
 def patch_cluster(cluster):
     """Patch a cluster for testing."""
+    cluster.PLUGGED_ATTR_READS = {}
+
+    async def _read_attribute_raw(attributes, *args, **kwargs):
+        LOGGER.debug(
```

Comment:
We normally don't use the logger in tests.

Reply:
Removed in https://github.com/home-assistant/core/pull/43357/commits/12be645a3d4222056804ac51ca716d1b68cacffe